lib/generators/boring/database_cleaner/active_record/rspec/install/install_generator.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module Boring
+  module DatabaseCleaner
+    module ActiveRecord
+      module Rspec
+        class InstallGenerator < Rails::Generators::Base
+          desc 'Adds database_cleaner-active_record gem to the application for cleaning ActiveRecord databases using different strategies'
+
+          TEST_FRAMEWORK = 'rspec-rails'
+
+          def add_database_cleaner_active_record_gem
+            say 'Adding database_cleaner-active_record gem', :green
+
+            Bundler.with_unbundled_env do
+              run "bundle add database_cleaner-active_record --group='test'"
+            end
+          end
+
+          def configure_database_cleaner
+            verify_presence_of_rspec_gem
+
+            create_file 'spec/support/database_cleaner.rb' do
+              rspec_config
+            end
+            inject_into_file 'spec/rails_helper.rb',
+                             "require 'spec/support/database_cleaner.rb'\n\n",
+                             before: /\A/
+          end
+
+          private
+
+          def rspec_config
+            <<~RUBY
+              RSpec.configure do |config|
+                config.before(:suite) do
+                  DatabaseCleaner.clean_with(:truncation)
+                end
+
+                config.before do
+                  DatabaseCleaner.strategy = :transaction
+                end
+
+                config.before(:each, js: true) do
+                  DatabaseCleaner.strategy = :truncation
+                end
+
+                config.before do
+                  DatabaseCleaner.start
+                end
+
+                config.after do
+                  DatabaseCleaner.clean
+                end
```

Comment:
Let's also research correct configuration for all type of tests and make sure we have added them here. Once we have done that, we can also provide users some option to skip or add configurations as needed using class_options.

Examples for class_options can also be found easily in most of the generators in this repo.

Reply:
It appears suitable for all types of tests, although several articles suggest setting `config.use_transactional_fixtures` to false instead of true. Additionally, we could introduce a specific configuration for feature tests, allowing both JS and feature tests to be skipped if necessary. @coolprobn dai, I'd like to hear your thoughts on this, particularly regarding `config.use_transactional_fixtures`.

Reply:
I agree, let's create options to skip configs as needed e.g. skip_capybara_configs skip_js_configs, etc.. 

By default we can add configurations for all of them